### DISCUSSION TITLE
Shared validation rules storage

### DISCUSSION THREAD

**Discussion by d1rOn** [directus/directus#17070] [Feature Requests]
It would be nice to have possibility to define some validation rules once and use them whenever it need.

It can be one more page in settings module, where we can predefine rule, give a name to it.

Currently we need to copy rules every time, it much less convenient.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
